.github/workflows/assets.yml

```diff
@@ -83,15 +83,9 @@ jobs:
       - name: Rename files for release
         id: rename-files-for-release
         run: |
-          mv assets/rootfs.img assets/${{ env.ARCH }}.rootfs.img
-          mv assets/kernel assets/${{ env.ARCH }}.kernel
-          mv assets/installer.img assets/${{ env.ARCH }}.installer.img
-          mv assets/initrd.img assets/${{ env.ARCH }}.initrd.img
-          mv assets/initrd.bits assets/${{ env.ARCH }}.initrd.bits
-          mv assets/ipxe.efi assets/${{ env.ARCH }}.ipxe.efi
-          mv assets/ipxe.efi.cfg assets/${{ env.ARCH }}.ipxe.efi.cfg
-          mv assets/ipxe.efi.ip.cfg assets/${{ env.ARCH }}.ipxe.efi.ip.cfg
-          mv assets/collected_sources.tar.gz assets/${{ env.ARCH }}.collected_sources.tar.gz
+            for asset in assets/*; do
```

Comment:
The `for` loop is currently indented with 4 spaces. Using 2 spaces would be more appropriate, in line with WF's standard indentation practice.

Reply:
Good catch. Done.